runtime/precompiles/src/JoinCandidatesWrapper.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-3.0-only.
+
+pragma solidity >=0.8.0;
+import "./StakingInterface.sol";
+
+/// An even more dead simple example to call the precompile
+contract JoinCandidatesWrapper {
+    /// The ParachainStaking wrapper at the known pre-compile address. This will be used to make
+    /// all calls to the underlying staking solution
+    ParachainStaking public staking;
+
+    /// Solely for debugging purposes
+    event Trace(uint256);
+
+    constructor(address _staking) {
+        staking = ParachainStaking(_staking);
+    }
+
+    receive() external payable {}
+
+    function join() public {
+        emit Trace(1 << 248); // Shift them to the left-most byte so I can see them in Apps
```

Comment:
This Events are intentional?

Reply:
This file will be removed as per Joshy's comments further above, but you're right the events were only there for debugging purposes.